lib/bugsnag/resque.rb

```diff
@@ -37,7 +37,7 @@ def save
 # Auto-load the failure backend
 Bugsnag::Resque.add_failure_backend
 
-Resque.before_first_fork do
+Resque.after_fork do
   Bugsnag.configuration.app_type = "resque"
   Bugsnag.configuration.delivery_method = :synchronous
```

Comment:
Why would you want to make a change? Just trying to understand the use case. Certain values of this will prevent you being notified so I just want to be careful with this.